apps/project/schema.py

```diff
@@ -424,7 +423,6 @@ class Meta:
     recent_active_users = graphene.List(graphene.NonNull(UserEntityDateType))
     top_sourcers = graphene.List(graphene.NonNull(UserEntityCountType))
     top_taggers = graphene.List(graphene.NonNull(UserEntityCountType))
-    assessment_reg_options = graphene.Field(AssessmentRegistryOptionsType)
```

Comment:
Not required anymore? Or we have moved this?

Reply:
@thenav56 , this is removed in the next pr https://github.com/the-deep/server/pull/1324